app.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express')
+const app = express()
+
+const { getAllTopics } = require('./controllers/topics.controllers.js')
+const {handleCustomErrors} = require('./errors/errors.js')
```

Comment:
I guess this is set up work in advance but we really should only be adding code we are using for the particular task. 🙂